pymc3_models/models/LinearRegression.py

```diff
@@ -93,17 +93,17 @@ def fit(self, X, y, inference_type='advi', minibatch_size=None, inference_args=N
 
         return self
 
-    def predict(self, X, return_std=False):
+
+    def sample(self, X, samples=2000):
         """
-        Predicts values of new data with a trained Linear Regression model
+        samples the conditional posterior estimates
 
         Parameters
         ----------
         X : numpy array, shape [n_samples, n_features]
 
-        return_std : Boolean flag of whether to return standard deviations with mean values. Defaults to False.
+        samples : number of draws to make for each point
```

Comment:
Could you add in the default value to this docstring? Like I do in the predict method below with `return_std`.